scripts/changeset.sh

```
@@ -0,0 +1,320 @@
+
+# Get current date and time
+DATE=$(date +%Y.%m.%d)
+TIME=$(date +%H.%M)
+
+# Get current branch name
+CURRENT_BRANCH=$(git branch --show-current)
+
+# Get last commit and previous commit hashes
+LAST_COMMIT=$(git rev-parse --short HEAD)
+PREVIOUS_COMMIT=$(git rev-parse --short HEAD~1 2>/dev/null || echo "none")
+LAST_COMMIT_FULL=$(git rev-parse HEAD)
```

Shellcheck (0.11.0)

[warning] LAST_COMMIT_FULL appears unused. Verify use (or export if used externally).

(SC2034)